src/main/kotlin/dev/stalla/model/atom/Link.kt

```diff
@@ -8,13 +8,18 @@ import dev.stalla.model.MediaType
 /**
  * Model class for elements describing hyperlinks.
  *
- * @property href The href of the link.
- * @property hrefLang The hrefLang of the link.
- * @property hrefResolved The hrefResolved of the link.
- * @property length The length of the link.
- * @property rel The rel of the link.
- * @property title The title of the link.
- * @property type The type of the link.
+ * Direct instantiation in Java is discouraged. Use the [builder][Link.Factory.builder]
+ * method to obtain a [LinkBuilder] instance for expressive construction instead.
+ *
+ * @property href The `href` attribute textContent of the `<atom:link>` element.
```

Comment:
@mpgirro this needs an answer/action

Reply:
Sorry, oversaw this. 

I agree. But I think `content` is ambiguous. The content of an XML element can be other elements. Technically, an elements text is a text node itself, and an element can have mixed content (multiple nested text nodes and other element nodes).

I suggest we refer to it as either the "attribute text" or the "element text". What do you think?

Reply:
You're right about the ambiguity. Your suggestions seem good to me.  A possible alternative may be "text content"? Should be universally valid (even though attrs can only have text content and it's thus a bit redundant, at least it's clear).

Reply:
I've pushed changes that incorporate both ideas by either referring to "element text content" or "attribute text content". We already had lots of "field text content" comments around, so its kind of the natural conclusion.

Another thing I've noticed though is that we also have property docs that don't refer to the backing XML construct at all, but instead give a good description about the intended purpose of the property. E.g.:

```
@property url The URL where listeners can find funding information for the podcast.
```

or 

```
@property owner An email address that can be used to verify ownership when moving hosting platforms.
``` 

It's quite a hard contrast when you compare those to e.g.:

```
@property hrefResolved The `hrefResolved` attribute text content of the `<atom:link>` element.
```

where users still have to figure out themselves what the property is for. And I mean, who except us really has a look at the standards anyway.

What do you think? Which approach is generally better. As a library user, I think I'd either prefer the seconds approach, or a combination where the technical information ("`hrefResolved` attribute text content of the `<atom:link>` element") is appended.

Reply:
It would certainly be better if we expressed the specs in the KDoc. Ideally, we'd also include info about the max theoretically allowed length for text fields (it changes a lot between various fields, I have found). Definitely not something for this PR though. Also probably something we may consider for a dot release such as 1.0.1, if the docs are the only thing that improves.

Reply:
I agree, we can always polish the docs later on.